src/core/tritonserver.cc

```diff
@@ -43,6 +41,7 @@
 #include "src/core/server.h"
 #include "src/core/server_message.h"
 #include "src/core/status.h"
+#include "src/core/tritonserver_apis.h"
```

Comment:
Why are we creating a new file instead of using tritonserver.h? Files that do not need tritonbackend.h end up including it as well.

Reply:
tritonserver.h (and tritonbackend.h) are used with different declations (either import or export of the api). tritonserver_apis.h sets defines that cause the API to be exported from libtritonserver.so. Direct uses of tritonserver.h and tritonbackend.h are normal client uses that "import" the API. All uses to the headers should use tritonserver_api.h with "core" triton. I combined them into 1 header for convenience.